pkg/network/protocols/http/ebpf_javatls.go

```diff
@@ -78,7 +103,37 @@ func (p *JavaTLSProgram) GetAllUndefinedProbes() (probeList []manager.ProbeIdent
 	return
 }
 
+func allowBlockAttachment(pid uint32) (reject bool) {
```

Comment:
A few comments here:
- The `reject` variable is not used
- Maybe add a comment explaining on what criterions the attachment is allowed or not.
- I am not sure the name `allowBlockAttachment` conveys what the function really does i.e returning `true` if the attachment is blocked. How about something like `isAttachmentAllowed` (would need to invert the return values so it returns `true` if it is allowed), making the usage look like this in the function below:
```go
if !isAttachmentAllowed(pid) {
```
WDYT?

Reply:
thanks `if !isAttachmentAllowed()` logic / is a better name